Frontend/src/Components/Dashboard/Dashboard.js

```diff
@@ -12,7 +12,7 @@ import DriverComms from "../Communication/DriverComms";
 import Communication from "../Communication/Communication";
 import IOView from "../Communication/IOView";
 import BatteryCells from "../BatteryCells/BatteryCells";
-import BatteryPack from "../BatteryCells/BatteryPack";
+import Battery from "../Battery/Battery";

```

Comment:
```suggestion
import BatteryPack from "../BatteryPack/BatteryPack";
```
Goes with the previous comments (and adds an additional nit-picky change):
- The component should be renamed `BatteryPack` so that it can naturally replace the previous one.
- Changing the file name to match the component name.
- The directory name doesn't _need_ to change, but to be more specific and differentiate it from the `BatteryCells` directory, I think it would be good to rename the directory to "BatteryPack"